### DISCUSSION TITLE
Building from Source Failed

### DISCUSSION THREAD

**Discussion by op90786** [keycloak/keycloak#12665] [Q&A]
Hi everyone,

I am trying to build Keycloak locally from source , following steps mentioned over here.
https://github.com/keycloak/keycloak/blob/main/docs/building.md
but after running `mvn clean install` 
I am getting some errors attached SS.

Looking forward to you guidance and help

![image](https://user-images.githubusercontent.com/107326024/175166879-4f975aff-8f89-4e0b-b676-8b4f91b75bb5.png)
![image](https://user-images.githubusercontent.com/107326024/175166940-0b47dcd5-24c4-46b3-b71f-e0090d48af9f.png)

**Comment by abstractj** [comment]
@op90786 could you please provide your JDK version and OS?
